tests/foreman/conftest.py

```diff
@@ -389,3 +396,77 @@ def module_gce_compute(module_org, module_location):
         zone=settings.gce.zone, organization=[module_org],
         location=[module_location]).create()
     return gce_cr
+
+
+@pytest.fixture(scope='module')
```

Comment:
session scope? We are not going to change the properties AFAIK in any tiers / test.

Reply:
good catch .. we could.